### DISCUSSION TITLE
When will new helm chart be released?

### DISCUSSION THREAD

**Discussion by Sh1ftry** [strimzi/strimzi-kafka-operator#4475] [Q&A]
Currently `0.21.1` is one month old and is not working with latest operator image, because of missing image for `2.6.1` in `STRIMZI_KAFKA_IMAGES`. I need to use latest operator image, because of https://github.com/strimzi/strimzi-kafka-operator/issues/4331. Do you know when will new helm chart be relesed?

Or maybe it would be possible to release the docker image under some dedicated tag with a fix for the certificates issue, but not requiring `2.6.1` in `STRIMZI_KAFKA_IMAGES`?

**Comment by scholzj** [comment]
The Helm Chart is released as part of a Strimzi release - Helm Chart for Strimzi 0.21.1 is expected to work with 0.21.1 Kafka images. The _latest_ images are master builds which change all the time and cannot be used for anything beyond testing and development. The current plan is to have first RC for Strimzi 0.22 next week.

You should be able to workaround the #4331 by specifying the certificate in the Kafka CR - you can just get the public key of the CA which signed your certificates. I know it is not as easy as using the JVM certificates directly. But it should be fairly easy as a workaround until 0.22 is released.
